docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -573,6 +581,132 @@ To receive the reply of processing command on the server side:
 > This feature supports Tizen 4.0 and Higher for Mobile.
 
 
+## Sending and Processing a Search Command
+
+To send a search command to the client from the server side:
+
+1. Create the media controller client handle using the `mc_client_create()`:
+
+   ```
+   ret = mc_client_create(&g_client_h);
+   ```
+
+2. Retrieve the server name using the `mc_client_foreach_server()`:
+
+   ```
+   bool
+   activated_server_cb(const char *server_name, void *user_data)
+   {
+     GList **server_list = (GList **)user_data;
+
+     if (!server_list || !server_name) return FALSE;
+     *server_list = g_list_append(*client_list, g_strdup(server_name));
+
+     return TRUE;
+   }
+
+   GList *server_list = NULL;
+
+   ret = mc_client_foreach_server(g_client_h, activated_server_cb, &server_list);
+   ```
+
+3. Create the search handle using the `mc_search_create()`:
+
+   ```
+   ret = mc_search_create(&g_search_h);
+   ```
+
+4. Set the condition with [a content type](#media-controller-content-type-attributes), [a search category](#media-controller-search-category-attributes) and a search keyword using the `mc_search_set_condition()`:
```

Comment:
I just meant to change the hyper linked words. 
For now "**`a content type`**" and "**`a search category`**" are used to set hyper link. 
My suggestion was to create hyper link only to "content" and  "search ", 
among "a **`content`** type" and "a **`search`** category", remaining others same.

Would you please check my suggestion? If you think the original is better, you can keep it.

Reply:
Okay, it's done. could you please check it?
After it is ok, I will be ready for updating it to master branch.

Reply:
Sorry for the late answer.

The changed I asked was on line 619, from
 -  `[a content type](#media-controller-content-type-attributes)` : "[a content type](#media-controller-content-type-attributes)"
 -  `[a search category](#media-controller-search-category-attributes)` : "[a search category](#media-controller-search-category-attributes)"

to 
 - `a [content](#media-controller-content-type-attributes) type` : "a [content](#media-controller-content-type-attributes) type"
 - `a [search](#media-controller-search-category-attributes) category` : "a [search](#media-controller-search-category-attributes) category"

From 3 hyperlinked words (a content type), to 1 hyperlinked word (content).


However, if you want to keep the hyper link to every 3 words, let's keep it as it is.


The merging tizen_5.0_prepare branch to master is done.
So you can apply this fix to master branch now.
Would you please apply this fix to master branch?
I'll accept that PR and close this one.

Reply:
I'm sorry that I misunderstood your comment.
I changed 3 hyperlinked words to 2 hyperlinked words. Because I think that two words like content-type and search-category have more meaning for the API.
And I updated this fix with changing hyperlinked words to master branch. Please refer to below link.
https://github.com/Samsung/tizen-docs/pull/468